plugins/jira/skills/status-analysis/scripts/gather_status_data.py

```
@@ -715,6 +729,65 @@ async def get_prs_batch(
+                try:
+                    async with session.post(
+                        self.GRAPHQL_URL,
+                        headers=self._get_headers(),
+                        json={"query": query},
+                        timeout=aiohttp.ClientTimeout(total=30),
+                    ) as resp:
+                        if resp.status == 200:
+                            data = await resp.json()
+                        else:
+                            logger.warning(f"  Lightweight batch {batch_num}/{total_batches}: HTTP {resp.status}")
+                except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)